paddle/fluid/inference/analysis/ir_pass_manager.cc

```diff
@@ -62,6 +65,11 @@ void IRPassManager::CreatePasses(Argument *argument,
                     argument->mkldnn_enabled_op_types()));
     }
 
+    if (pass_name == "anakin_subgraph_pass") {
+      pass->Set("program",
+                new framework::ProgramDesc *(&argument->main_program()));
```

Comment:
the main_program is the original program,
it is also stored in the graph passed to Graph.